packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -163,7 +163,7 @@ export interface AutocompleteProps<
    */
   ListboxProps?: ReturnType<ReturnType<typeof useAutocomplete>['getListboxProps']> & {
     sx?: SxProps<Theme>;
-  };
+  } & { ref?: React.Ref<any> };
```

Comment:
This should work as well:

```suggestion
  } & { ref?: React.Ref<Element> };
```